service/awsconfig/v4/key/key.go

```diff
@@ -2,11 +2,15 @@ package key
 
 import (
 	"fmt"
-	"path/filepath"
 	"strings"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/aws-operator/service/awsconfig/v4/templates/cloudconfig"
+	"github.com/giantswarm/aws-operator/service/awsconfig/v4/templates/cloudformation/guest"
+	"github.com/giantswarm/aws-operator/service/awsconfig/v4/templates/cloudformation/hostpost"
+	"github.com/giantswarm/aws-operator/service/awsconfig/v4/templates/cloudformation/hostpre"
```

Comment:
We have the templates as constants now and thus can import them and version them easily with the rest of our code.